### PR TITLE
Program: Display the crash dialog in Release mode

### PR DESCRIPTION
**Pull request by lioncash**
Makes crash reporting from users easier, as they can now indicate (and moreover, actually see) what the reason for a crash is.

### CODE REVIEW

File: LiveSplit/LiveSplit/Program.cs [hunk 1]
```diff
@@ -39,15 +39,10 @@ static void Main(string[] args)
                 if (Twitch.Instance != null)
                     Twitch.Instance.CloseAllChatConnections();
             }
-#if !DEBUG
```
Comment (wooferzfg1) [inline LiveSplit/LiveSplit/Program.cs]: This was !DEBUG (so Release only), which is what we want so that Visual Studio shows the error in Debug mode.

Comment (lioncash) [inline LiveSplit/LiveSplit/Program.cs]: tfw I can't read

Comment (lioncash) [inline LiveSplit/LiveSplit/Program.cs]: I'll update it to simply include the exception message, however, gimme a sec.

**Comment by wooferzfg1** [comment]
We definitely want to display what the exception is, though. That was a cool change.

**Pull request closed without merging**